db/migrate/20220326183004_likes.rb

```diff
@@ -0,0 +1,9 @@
+class Likes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :Likes do |t|
+      t.references :User, null: false, foreign_key: true, index: true
+      t.references :Post, null: false, foreign_key: true, index: true
```

Comment:
- [ ] <b>REQUIRED:</b> Remember for User, the project requires to use `author_id`. You can check it out in the given [ERD Diagram.](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png) so we do it like this
```rb
t.references :author, foreign_key: { to_table: 'users' } 
```
- [ ] <b>REQUIRED: And also, please user the `lower case` and `snake case`.